src/lib/lines/VerticalLines.js

```diff
@@ -53,12 +56,18 @@ export default class VerticalLines extends Component {
           Math.ceil((nextTime.valueOf() - time.valueOf()) * ratio) - lineWidth
         const leftPush = firstOfType ? -1 : 0
 
+        let classNamesForTime = []
+        if (verticalLineClassNamesForTime) {
+          classNamesForTime = [verticalLineClassNamesForTime(time.unix() * 1000, nextTime.unix() * 1000 - 1)];
```

Comment:
Can you explain this line really quick:

* why `* 1000`?
* why subtract 1 from the `nextTime`?